pkgs/development/libraries/amtk/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl
+, pkgconfig, gnome3, dbus, xvfb_run }:
+let
+  version = "4.99.1";
+  pname = "amtk";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "00fhvw5y638z584s8cfdslh47ngfzcgx4f0b0456sw8p754j3f8d";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    dbus.daemon
+    gnome3.gtk.dev
```

Comment:
Why specifically request dev output?

Reply:
That's what I found when looking for the missing file using `nix-locate`

Reply:
I think we usually do not use `dev` outputs, since this will probably also use a library that will be in a different output (`out`). It works because the builder somehow inserts other outputs as well or something.